shared-module/displayio/display_core.c

```diff
@@ -366,7 +359,10 @@ void displayio_display_core_collect_ptrs(displayio_display_core_t *self) {
 }
 
 bool displayio_display_core_fill_area(displayio_display_core_t *self, displayio_area_t *area, uint32_t *mask, uint32_t *buffer) {
-    return displayio_group_fill_area(self->current_group, &self->colorspace, area, mask, buffer);
+    if (self->current_group != NULL) {
+        return displayio_group_fill_area(self->current_group,&self->colorspace, area, mask, buffer);
+    }
```

Comment:
It turned out that we needed an additional check for NULL internally to prevent the fill_area code from trying to access properties on a null object which was leading to hard fault. 

It was making it to here inside of Group fill_area: https://github.com/adafruit/circuitpython/blob/1f3b2433c04be44dfb325c7300a83f9efc26e894/shared-module/displayio/Group.c#L377 then having the hard fault. 

I added the check to fix it here in display_core instead of inside Group. It made more sense to me for the display to notice it's group is null and just do nothing, rather than blindly call into Group fill_area and have it be responsible for checking if self is NULL inside that function.